.gitlab-ci.yml

```diff
@@ -104,6 +113,15 @@ test_macos11_py37_coremltools_test:
   variables:
     WHEEL_PATH: build/dist/*cp37*10_16*
     PYTHON: "3.7"
+  only:
```

Comment:
@1duo what happens if changes are not under current filters? no tests are run? can we configure if no checks are met we test all?

Reply:
If files/dirs are not on this list, then the current test (with this condition) won't be triggered. So we should list as much as possible. Alternatively, if we can trigger tests when we change anything except a list of things.